madmom/features/tempo.py

```diff
@@ -310,6 +335,64 @@ def process(self, activations, **kwargs):
         # detect the tempi and return them
         return detect_tempo(histogram, self.fps)
 
+    def process_online(self, activations, reset=True, **kwargs):
+        """
+        Detect the tempi from the (beat) activations in online mode.
+
+        Parameters
+        ----------
+        activations : numpy array
+            For the online setting there will be only one activation at a time
+            but it is also possible to use it to process offline data.
+
+        Returns
+        -------
+        tempi : numpy array
+            Array with the dominant tempi [bpm] (first column) and their
+            relative strengths (second column).
+
+        """
+        # make the activations a 1D array
+        activations = np.atleast_1d(activations)
```

Comment:
Ah ok, I removed it now and adapted the tests (like you did for #299) here:
25a7522eda79907cdb07e5945214b89449f803b3